settings.py

```diff
@@ -7,7 +7,7 @@
 PLAYER_SPEED = 5
 POWER_UP_RATE = 200    # Lower number means more frequent power-ups
 SCORE_MULTIPLIER = 2   # For X2-Boost power-up
-ENEMY_SPEED = 1
+ENEMY_SPEED = 2 # Speed sollte immer int sein, sonst kommt es zu Problemen, da rect.x float nicht benutzen kann -> Rundungsfehler; komische Geschwindigkeiten
```

Comment:
Wäre es nicht vielleicht besser die Variable STANDARD_ENEMY_SPEED zu nennen? Sie müsste natürlich auch überall wo sie verwendet wird geändert werden. Würde es lesbarer machen. Wäre aber noch kurz Aufwand und es funktioniert ja.
```suggestion
STANDARD_ENEMY_SPEED = 2 # Speed sollte immer int sein, sonst kommt es zu Problemen, da rect.x float nicht benutzen kann -> Rundungsfehler; komische Geschwindigkeiten
```